### PR TITLE
Some machine updates for cascade at PNNL

### PR DESCRIPTION
**Pull request by singhbalwinder**
Updates for mach files after a recent machine outage

[BFB] - Bit-For-Bit

### CODE REVIEW

File: cime/config/e3sm/machines/config_batch.xml [hunk 1]
```diff
@@ -374,7 +374,9 @@
 	<directive>--error=slurm.err</directive>
       </directives>
       <queues>
-	<queue walltimemax="00:30:00" default="true">large</queue>
+	<queue walltimemax="00:59:00" nodemin="1"  nodemax="15"  >small</queue>
+	<queue walltimemax="00:59:00" nodemin="16" nodemax="127" >medium</queue>
```
[the remark this answers is not in the document]
Comment (singhbalwinder) [reply cime/config/e3sm/machines/config_batch.xml]: This is how understand how this machine works. It select queues based on number of nodes asked. For example, if you are asking for 1 node, it has to be in "small" queue and if you are asking for 18 nodes, it has to be in the "medium" queue. It doesn't allow, for example, a 1 node job to select "medium" queue (it throws an error for such cases). So, I think, "default" here doesn't do anything. Please let me know if "default" has a different meaning here.

[the remark this answers is not in the document]
Comment (jgfouca) [reply cime/config/e3sm/machines/config_batch.xml]: It's probably fine. There are cases where the default will be selected if none of the queues match.

[the remark this answers is not in the document]
Comment (singhbalwinder) [reply cime/config/e3sm/machines/config_batch.xml]: As far as I know, this machine doesn't let you decide which queue to go to. If you pick up a wrong queue ( or a default queue which happens to be wrong) , the job exits with error "invalid node configuration" or something similar (can't remember the exact error).

[the remark this answers is not in the document]
Comment (jgfouca) [reply cime/config/e3sm/machines/config_batch.xml]: OK, I approve.

[the remark this answers is not in the document]
Comment (singhbalwinder) [reply cime/config/e3sm/machines/config_batch.xml]: Thanks!

**Pull request merged**